sql/moz-fx-data-shared-prod/search_terms/quicksuggest_impression/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+---
+friendly_name: Historical Pings for `contextual-services/quicksuggest-impression`
+description: |-
+  A historical view of pings sent for the
+  `contextual-services/quicksuggest-impression`
+  document type, including search terms fields.
+
+  For search terms data contained in `quicksuggest-impression` pings,
+  consult the `search_terms` dataset which is governed by a separate access policy.
+
+  This view is guaranteed to contain only complete days
+  (per `submission_timestamp`)
+  and to contain only one row per distinct `document_id` within a given date.
+
+  Clustering fields: `normalized_channel`, `sample_id`
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:search-terms/sanitized
```

Comment:
This group doesn't currently exist. It's proposed in https://mozilla-hub.atlassian.net/browse/DSRE-151